packages/slice-machine/src/theme.ts

```diff
@@ -643,17 +643,38 @@ const AppTheme = (): Theme =>
       },
       table: {
         width: "100%",
-        my: 4,
-        borderCollapse: "separate",
+        borderCollapse: "collapse",
         borderSpacing: 0,
-        "th,td": {
+        "td, th": {
           textAlign: "left",
-          py: "4px",
-          pr: "4px",
-          pl: 0,
           borderColor: "muted",
           borderBottomStyle: "solid",
         },
+        td: {
```

Comment:
wouldn't that affect the slice list since they're both grid/table ?

Reply:
The slice list is not using the table HTML tags